ccl/multiregionccl: TestMrSystemDatabase failed

ccl/multiregionccl.TestMrSystemDatabase [failed](https://tanzanite.cluster.engflow.com/invocation/817e8f2d-e4dc-4b32-bfbb-783ac3877f53) on master @ [59a69ae0c5474a8b323afac269055bf6ce8afa34](https://github.com/cockroachdb/cockroach/commits/59a69ae0c5474a8b323afac269055bf6ce8afa34):


```
=== RUN   TestMrSystemDatabase
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestMrSystemDatabase3932306887
    test_log_scope.go:81: use -show-logs to present logs inline

pkg/ccl/multiregionccl/multiregion_system_table_test.go:60: (TestMrSystemDatabase)
	NOTICE: .Locality() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Locality() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    multiregion_system_table_test.go:250: -- test log scope end --
test logs left over in: outputs.zip/logTestMrSystemDatabase3932306887
--- FAIL: TestMrSystemDatabase (7.29s)
=== RUN   TestMrSystemDatabase/QueryByEnum
    multiregion_system_table_test.go:247: error scanning '&{0xc004986120 <nil>}': pq: unsupported comparison: bytes to crdb_internal_region
    --- FAIL: TestMrSystemDatabase/QueryByEnum (0.00s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=16</code>
, <code>shard=3</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMrSystemDatabase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-32590

ccl/multiregionccl.TestMrSystemDatabase [failed](https://tanzanite.cluster.engflow.com/invocation/be50d37f-c771-45ca-a89d-a7d3b521b6a5) on master @ [8315a4bc997fb8b8d679079e14d6d7ca94d53bc6](https://github.com/cockroachdb/cockroach/commits/8315a4bc997fb8b8d679079e14d6d7ca94d53bc6):


```
=== RUN   TestMrSystemDatabase
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestMrSystemDatabase384682926
    test_log_scope.go:81: use -show-logs to present logs inline

pkg/ccl/multiregionccl/multiregion_system_table_test.go:60: (TestMrSystemDatabase)
	NOTICE: .Locality() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Locality() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    multiregion_system_table_test.go:250: -- test log scope end --
test logs left over in: outputs.zip/logTestMrSystemDatabase384682926
--- FAIL: TestMrSystemDatabase (5.63s)
=== RUN   TestMrSystemDatabase/QueryByEnum
    multiregion_system_table_test.go:247: error scanning '&{0xc0063f2360 <nil>}': pq: unsupported comparison: bytes to crdb_internal_region
    --- FAIL: TestMrSystemDatabase/QueryByEnum (0.00s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=11</code>
, <code>shard=3</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMrSystemDatabase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/multiregionccl.TestMrSystemDatabase [failed](https://tanzanite.cluster.engflow.com/invocation/21152969-dbbb-4288-85cc-b5b02353c383) on master @ [cedd409f2115655efe2c529f6052f3056673b277](https://github.com/cockroachdb/cockroach/commits/cedd409f2115655efe2c529f6052f3056673b277):


```
=== RUN   TestMrSystemDatabase
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestMrSystemDatabase634443294
    test_log_scope.go:81: use -show-logs to present logs inline

pkg/ccl/multiregionccl/multiregion_system_table_test.go:60: (TestMrSystemDatabase)
	NOTICE: .Locality() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Locality() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    multiregion_system_table_test.go:250: -- test log scope end --
test logs left over in: outputs.zip/logTestMrSystemDatabase634443294
--- FAIL: TestMrSystemDatabase (8.28s)
=== RUN   TestMrSystemDatabase/QueryByEnum
    multiregion_system_table_test.go:247: error scanning '&{0xc006451d40 <nil>}': pq: unsupported comparison: bytes to crdb_internal_region
    --- FAIL: TestMrSystemDatabase/QueryByEnum (0.00s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=23</code>
, <code>shard=3</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMrSystemDatabase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/multiregionccl.TestMrSystemDatabase [failed](https://tanzanite.cluster.engflow.com/invocation/01d665e8-8598-4370-8b66-c313cf26b116) on master @ [d131065cc4ebb176c027892cac5cdb96cd69c214](https://github.com/cockroachdb/cockroach/commits/d131065cc4ebb176c027892cac5cdb96cd69c214):


```
=== RUN   TestMrSystemDatabase
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestMrSystemDatabase2290391886
    test_log_scope.go:81: use -show-logs to present logs inline

pkg/ccl/multiregionccl/multiregion_system_table_test.go:60: (TestMrSystemDatabase)
	NOTICE: .Locality() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Locality() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    multiregion_system_table_test.go:250: -- test log scope end --
test logs left over in: outputs.zip/logTestMrSystemDatabase2290391886
--- FAIL: TestMrSystemDatabase (7.29s)
=== RUN   TestMrSystemDatabase/QueryByEnum
    multiregion_system_table_test.go:247: error scanning '&{0xc00e79c360 <nil>}': pq: unsupported comparison: bytes to crdb_internal_region
    --- FAIL: TestMrSystemDatabase/QueryByEnum (0.00s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=12</code>
, <code>shard=3</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMrSystemDatabase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/multiregionccl.TestMrSystemDatabase [failed](https://tanzanite.cluster.engflow.com/invocation/17e905b9-37a7-43a3-82fa-f3ebd7bd394f) on master @ [414cb46513079bb8869bd776f045ed6c96ad6d26](https://github.com/cockroachdb/cockroach/commits/414cb46513079bb8869bd776f045ed6c96ad6d26):


```
=== RUN   TestMrSystemDatabase
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestMrSystemDatabase265023528
    test_log_scope.go:81: use -show-logs to present logs inline

pkg/ccl/multiregionccl/multiregion_system_table_test.go:60: (TestMrSystemDatabase)
	NOTICE: .Locality() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Locality() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    multiregion_system_table_test.go:250: -- test log scope end --
test logs left over in: outputs.zip/logTestMrSystemDatabase265023528
--- FAIL: TestMrSystemDatabase (7.27s)
=== RUN   TestMrSystemDatabase/QueryByEnum
    multiregion_system_table_test.go:247: error scanning '&{0xc00d74d7a0 <nil>}': pq: unsupported comparison: bytes to crdb_internal_region
    --- FAIL: TestMrSystemDatabase/QueryByEnum (0.00s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=8</code>
, <code>shard=3</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMrSystemDatabase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/multiregionccl.TestMrSystemDatabase [failed](https://tanzanite.cluster.engflow.com/invocation/fa25fbe3-5330-4b4d-9e82-5f36b16f3a0c) on master @ [fbb3446238434b494751af2fe699a4941e43fdd6](https://github.com/cockroachdb/cockroach/commits/fbb3446238434b494751af2fe699a4941e43fdd6):


```
=== RUN   TestMrSystemDatabase
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestMrSystemDatabase31665262
    test_log_scope.go:81: use -show-logs to present logs inline

pkg/ccl/multiregionccl/multiregion_system_table_test.go:60: (TestMrSystemDatabase)
	NOTICE: .Locality() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Locality() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    multiregion_system_table_test.go:250: -- test log scope end --
test logs left over in: outputs.zip/logTestMrSystemDatabase31665262
--- FAIL: TestMrSystemDatabase (7.31s)
=== RUN   TestMrSystemDatabase/QueryByEnum
    multiregion_system_table_test.go:247: error scanning '&{0xc00d822ea0 <nil>}': pq: unsupported comparison: bytes to crdb_internal_region
    --- FAIL: TestMrSystemDatabase/QueryByEnum (0.00s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=1</code>
, <code>shard=3</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMrSystemDatabase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/multiregionccl.TestMrSystemDatabase [failed](https://tanzanite.cluster.engflow.com/invocation/9c7074e7-2043-4df6-93ba-e96351b5a412) on master @ [14733120c08ba1a0a050bb5442c3311090206566](https://github.com/cockroachdb/cockroach/commits/14733120c08ba1a0a050bb5442c3311090206566):


```
=== RUN   TestMrSystemDatabase
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestMrSystemDatabase3351160154
    test_log_scope.go:81: use -show-logs to present logs inline

pkg/ccl/multiregionccl/multiregion_system_table_test.go:60: (TestMrSystemDatabase)
	NOTICE: .Locality() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Locality() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    multiregion_system_table_test.go:250: -- test log scope end --
test logs left over in: outputs.zip/logTestMrSystemDatabase3351160154
--- FAIL: TestMrSystemDatabase (7.28s)
=== RUN   TestMrSystemDatabase/QueryByEnum
    multiregion_system_table_test.go:247: error scanning '&{0xc00bf15e60 <nil>}': pq: unsupported comparison: bytes to crdb_internal_region
    --- FAIL: TestMrSystemDatabase/QueryByEnum (0.00s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=19</code>
, <code>shard=3</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMrSystemDatabase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/multiregionccl.TestMrSystemDatabase [failed](https://tanzanite.cluster.engflow.com/invocation/340c9760-de79-476a-a36e-c876a70d951b) on master @ [64dacab2808c5420e3fd99d2562e8ecb97b7742b](https://github.com/cockroachdb/cockroach/commits/64dacab2808c5420e3fd99d2562e8ecb97b7742b):


```
=== RUN   TestMrSystemDatabase
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestMrSystemDatabase979851441
    test_log_scope.go:81: use -show-logs to present logs inline

pkg/ccl/multiregionccl/multiregion_system_table_test.go:60: (TestMrSystemDatabase)
	NOTICE: .Locality() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Locality() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    multiregion_system_table_test.go:250: -- test log scope end --
test logs left over in: outputs.zip/logTestMrSystemDatabase979851441
--- FAIL: TestMrSystemDatabase (7.26s)
=== RUN   TestMrSystemDatabase/QueryByEnum
    multiregion_system_table_test.go:247: error scanning '&{0xc00f4b0480 <nil>}': pq: unsupported comparison: bytes to crdb_internal_region
    --- FAIL: TestMrSystemDatabase/QueryByEnum (0.00s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=15</code>
, <code>shard=3</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMrSystemDatabase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/multiregionccl.TestMrSystemDatabase [failed](https://tanzanite.cluster.engflow.com/invocation/9f917b34-099d-49ca-aa75-b4acb6f02fad) on master @ [2859e5fab431536ec77f646787af637eec3d185c](https://github.com/cockroachdb/cockroach/commits/2859e5fab431536ec77f646787af637eec3d185c):


```
=== RUN   TestMrSystemDatabase
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestMrSystemDatabase725305899
    test_log_scope.go:81: use -show-logs to present logs inline

pkg/ccl/multiregionccl/multiregion_system_table_test.go:60: (TestMrSystemDatabase)
	NOTICE: .Locality() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Locality() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    multiregion_system_table_test.go:250: -- test log scope end --
test logs left over in: outputs.zip/logTestMrSystemDatabase725305899
--- FAIL: TestMrSystemDatabase (7.28s)
=== RUN   TestMrSystemDatabase/QueryByEnum
    multiregion_system_table_test.go:247: error scanning '&{0xc002636480 <nil>}': pq: unsupported comparison: bytes to crdb_internal_region
    --- FAIL: TestMrSystemDatabase/QueryByEnum (0.00s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=14</code>
, <code>shard=3</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMrSystemDatabase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/multiregionccl.TestMrSystemDatabase [failed](https://tanzanite.cluster.engflow.com/invocation/3ef5e249-f8d6-4daf-a31c-5c34f4c0fb69) on master @ [a87cefce46dc064e0d782dbfde55e0a1f3db1abc](https://github.com/cockroachdb/cockroach/commits/a87cefce46dc064e0d782dbfde55e0a1f3db1abc):


```
=== RUN   TestMrSystemDatabase
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestMrSystemDatabase1361128240
    test_log_scope.go:81: use -show-logs to present logs inline

pkg/ccl/multiregionccl/multiregion_system_table_test.go:60: (TestMrSystemDatabase)
	NOTICE: .Locality() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Locality() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    multiregion_system_table_test.go:250: -- test log scope end --
test logs left over in: outputs.zip/logTestMrSystemDatabase1361128240
--- FAIL: TestMrSystemDatabase (5.52s)
=== RUN   TestMrSystemDatabase/QueryByEnum
    multiregion_system_table_test.go:247: error scanning '&{0xc00e6b99e0 <nil>}': pq: unsupported comparison: bytes to crdb_internal_region
    --- FAIL: TestMrSystemDatabase/QueryByEnum (0.00s)
```
<p>Parameters: <code>attempt=1</code>
, <code>run=21</code>
, <code>shard=3</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMrSystemDatabase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/multiregionccl.TestMrSystemDatabase [failed](https://tanzanite.cluster.engflow.com/invocation/4cf4e7c1-c4d9-4d1b-a99a-a24f2e381015) on master @ [37ad01a3972cb4d34bfc6dfb4b9cfcac360b15dd](https://github.com/cockroachdb/cockroach/commits/37ad01a3972cb4d34bfc6dfb4b9cfcac360b15dd):


```
=== RUN   TestMrSystemDatabase
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestMrSystemDatabase3466943959
    test_log_scope.go:81: use -show-logs to present logs inline

pkg/ccl/multiregionccl/multiregion_system_table_test.go:60: (TestMrSystemDatabase)
	NOTICE: .Locality() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Locality() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    multiregion_system_table_test.go:250: -- test log scope end --
test logs left over in: outputs.zip/logTestMrSystemDatabase3466943959
--- FAIL: TestMrSystemDatabase (5.91s)
=== RUN   TestMrSystemDatabase/QueryByEnum
    multiregion_system_table_test.go:247: error scanning '&{0xc006bb2d80 <nil>}': pq: unsupported comparison: bytes to crdb_internal_region
    --- FAIL: TestMrSystemDatabase/QueryByEnum (0.00s)
```
<p>Parameters: <code>=</code>
, <code>attempt=1</code>
, <code>run=11</code>
, <code>shard=3</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMrSystemDatabase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/multiregionccl.TestMrSystemDatabase [failed](https://tanzanite.cluster.engflow.com/invocations/default/153328c7-f4e4-4aef-b85c-9008aa1bc036%3FtestReportRun=4&testReportShard=3&testReportAttempt=1%23targets-Ly9wa2cvY2NsL211bHRpcmVnaW9uY2NsOm11bHRpcmVnaW9uY2NsX3Rlc3Q=) on master @ [c10199436fe510962ea623d7e1233d71ee8c4a5e](https://github.com/cockroachdb/cockroach/commits/c10199436fe510962ea623d7e1233d71ee8c4a5e):


```
=== RUN   TestMrSystemDatabase
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestMrSystemDatabase4145695538
    test_log_scope.go:81: use -show-logs to present logs inline

pkg/ccl/multiregionccl/multiregion_system_table_test.go:60: (TestMrSystemDatabase)
	NOTICE: .Locality() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Locality() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    multiregion_system_table_test.go:250: -- test log scope end --
test logs left over in: outputs.zip/logTestMrSystemDatabase4145695538
--- FAIL: TestMrSystemDatabase (5.92s)
=== RUN   TestMrSystemDatabase/QueryByEnum
    multiregion_system_table_test.go:247: error scanning '&{0xc00bc15c20 <nil>}': pq: unsupported comparison: bytes to crdb_internal_region
    --- FAIL: TestMrSystemDatabase/QueryByEnum (0.00s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=4</code>
 - <code>shard=3</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMrSystemDatabase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/multiregionccl.TestMrSystemDatabase [failed](https://tanzanite.cluster.engflow.com/invocations/default/42189060-1895-4e2e-8d10-65c3c95cad53?testReportRun=18&testReportShard=3&testReportAttempt=1#targets-Ly9wa2cvY2NsL211bHRpcmVnaW9uY2NsOm11bHRpcmVnaW9uY2NsX3Rlc3Q=) on master @ [5f76d93b2ad8d29c14c9b176753ec33cbf81e3fe](https://github.com/cockroachdb/cockroach/commits/5f76d93b2ad8d29c14c9b176753ec33cbf81e3fe):


```
=== RUN   TestMrSystemDatabase
I231219 07:19:00.873351 1 (gostd) rand.go:250  [-] 1  random seed: -349633443087715652
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestMrSystemDatabase2422938484
    test_log_scope.go:81: use -show-logs to present logs inline

pkg/ccl/multiregionccl/multiregion_system_table_test.go:60: (TestMrSystemDatabase)
	NOTICE: .Locality() called via implicit interface ApplicationLayerInterface;
HINT: consider using .ApplicationLayer().Locality() instead.
TIP: consider replacing the test server initialization from:
    ts, ... := serverutils.StartServer(t, ...)
    defer ts.Stopper().Stop(...)
to:
    srv, ... := serverutils.StartServer(t, ...)
    defer srv.Stopper().Stop(...)
    ts := srv.ApplicationLayer()

See also: https://go.crdb.dev/p/testserver-and-cluster-virtualization
    multiregion_system_table_test.go:250: -- test log scope end --
test logs left over in: outputs.zip/logTestMrSystemDatabase2422938484
--- FAIL: TestMrSystemDatabase (5.55s)
=== RUN   TestMrSystemDatabase/QueryByEnum
    multiregion_system_table_test.go:247: error scanning '&{0xc00d919440 <nil>}': pq: unsupported comparison: bytes to crdb_internal_region
    --- FAIL: TestMrSystemDatabase/QueryByEnum (0.00s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=18</code>
 - <code>shard=3</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMrSystemDatabase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/multiregionccl/cold_start_latency_test.go
+++ b/pkg/ccl/multiregionccl/cold_start_latency_test.go
@@ -46,8 +46,7 @@ import (
 func TestColdStartLatency(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-	skip.UnderRace(t, "too slow")
-	skip.UnderStress(t, "too slow")
+	skip.UnderDuress(t, "too slow")
 
 	// We'll need to make some per-node args to assign the different
 	// KV nodes to different regions and AZs. We'll want to do it to

--- a/pkg/ccl/multiregionccl/multiregion_system_table_test.go
+++ b/pkg/ccl/multiregionccl/multiregion_system_table_test.go
@@ -64,8 +64,13 @@ func TestMrSystemDatabase(t *testing.T) {
 	tDB := sqlutils.MakeSQLRunner(tenantSQL)
 
 	// Generate stats for system.sqlinstances. See the "QueryByEnum" test for
-	// details.
+	// details. Since stats are generated asynchronously, we also poll to make
+	// sure the stats are generated.
 	tDB.Exec(t, `ANALYZE system.sqlliveness;`)
+	tDB.CheckQueryResultsRetry(t,
+		`select count(*) from [show statistics for table system.sqlliveness] where column_names = '{crdb_region}';`,
+		[][]string{{"1"}},
+	)
 
 	tDB.Exec(t, `ALTER DATABASE system SET PRIMARY REGION "us-east1"`)
 	tDB.Exec(t, `ALTER DATABASE system ADD REGION "us-east2"`)

